components/transactions/TransactionItem.js

```diff
@@ -176,11 +176,10 @@ const TransactionItem = ({ displayActions, collective, transaction, onMutationSu
               </Span>
             </Container>
             {hasOrder && (
-              <OrderStatusTag
-                status={order.status}
-                isRefund={isRefunded}
-                isReject={isRejected}
-                isRejectedRefundTransaction={isRefund && isRejected}
+              <TransactionStatusTag
+                isRefund={isRefund}
+                isRefunded={isRefunded}
+                isOrderRejected={isRefunded && isRejected}
```

Comment:
I think this would look clearer:

```suggestion
                isOrderRejected={isRejected}
```

Then in TransactionStatusTag.js do `isRefunded && isOrderRejected` instead of `isOrderRejected`.